features/com.google.gcp.eclipse.suite.e45.feature/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding/<project>=UTF-8
```

Comment:
It does look strange, but yes.  You can change the encoding of particular files which would be recorded as `encoding//path/in/project=XXX`